src/createStore.js

```diff
@@ -66,9 +66,13 @@ export default function createStore(reducer, initialState) {
         return
       }
 
-      isSubscribed = false
-      var index = listeners.indexOf(listener)
-      listeners.splice(index, 1)
+      listeners = listeners.filter(x => {
```

Comment:
Any particular reason why you're filtering rather than concat()ing two slice()d arrays and using indexOf() like before?